galaxy_ng/app/management/commands/set-repo-keyring.py

```diff
@@ -83,14 +90,14 @@ def handle(self, *args, **options):
             self.echo(f"Task failed with error: {task.error}", self.style.ERROR)
             sys.exit(1)
 
-        if AnsibleRepository.objects.get(pk=repo.pk).keyring == keyring:
+        if AnsibleRepository.objects.get(pk=repo.pk).gpgkey == pubkey:
             self.echo(f"Keyring {keyring} set successfully to {repository} repository")
 
 
-def set_repo_keyring(repo_id, keyring):
+def set_repo_gpgkey(repo_id, gpgkey):
```

Comment:
(I assume renaming keyring to gpgkey here is just for clarity / semantics) - taking in that public key rather than a the filepath, then assigning the gpgkey field in the repo table to that public key.

Reply:
Yes, it's a different string that is handled the same way.